### PR TITLE
Collection Display

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by victorg1991** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by victorg1991** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/fragments/components/CollectionDisplay.js [hunk 1]
```diff
@@ -14,11 +14,53 @@
...
+		},
+		item: {
+			name: Liferay.Language.get('collection-display'),
+			type: LAYOUT_DATA_ITEM_TYPES.collection,
+		},
+	});
+
+	useEffect(() => {
+		preview(getEmptyImage(), {captureDraggingState: true});
+	}, [preview]);
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/fragments/components/CollectionDisplay.js]: I think this drag stuff is repeated in 3 places right? Maybe is time to move that into a hook ? 😛

Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/fragments/components/CollectionDisplay.js]: Or an util, useUtil('preview')

Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/fragments/components/CollectionDisplay.js]: xDDD

Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/fragments/components/CollectionDisplay.js]: Me gusta mucho ese hook, que cargue fincheros de utils con el loader y te los devuelva

**Pull request closed without merging**